src/stitching/typeFromAST.ts

```diff
@@ -200,3 +206,18 @@ function createNamedStub(
     },
   });
 }
+
+function makeDirective(node: DirectiveDefinitionNode): GraphQLDirective {
+  const locations: Array<DirectiveLocationEnum> = [];
```

Comment:
I know this is just a style thing, but what do you think about: 
```ts
const locations: Array<DirectiveLocationEnum> = node.locations
    .filter(loc => <DirectiveLocationEnum>loc.value in DirectiveLocation)
    .map(loc => <DirectiveLocationEnum>loc.value);
```

Reply:
I really prefer for loops :)